src/commands/screen.js

```diff
@@ -0,0 +1,29 @@
+import { ensureInExtensionDir, loadExtensionJsonAsync } from '../extension/data';
+import * as shortcut from '../extension/shortcut';
+import { ensureVariableName } from '../extension/cli-parsing';
+import msg from '../user_messages';
+import { instantiateTemplatePath } from '../extension/template';
+import { createShortcutForScreen } from './shortcut';
+
+export async function createScreen(screenName, shortcutName = null) {
+  [screenName, shortcutName].map(ensureVariableName);
+  const extJson = await loadExtensionJsonAsync();
+
+  const shortcutData = shortcutName ? await shortcut.promptShortcutInfo(shortcutName) : null;
```

Comment:
There's very little to gain by parallelization as the promptShortcutInfo() function is awaiting for user's input.
I like to avoid parallelization if it isn't necessary because it (usually) reduces readability.